sources/packages/backend/apps/api/src/route-controllers/education-program/education-program.institutions.controller.ts

```diff
@@ -94,7 +94,10 @@ export class EducationProgramInstitutionsController extends BaseController {
    * @param payload information to be updated.
    */
   @ApiUnprocessableEntityResponse({
-    description: "Not able to a save the program due to an invalid request.",
+    description:
```

Comment:
Same error messages must be added here as well.

Reply:
Added the "duplicate SABC code." to the message since the other ones are related only to the update.
Please let me know if that was the expectation.